Task7/src/index.js

```diff
@@ -0,0 +1,183 @@
+        return new jQuery( jQuery.children(this._string, selector));
+    }
+
+    //Get the children of each element in the set of matched elements, optionally filtered by a selector.
+    static children(node, selector){
+        let element = document.querySelector(node);
+        let result = [];
+        if (!selector)
+            return element.childNodes;
+
+        for(let i = 0; i < element.childNodes.length; i++ ){            
+            if (element.childNodes[i].nodeType == 1 && 
+                (element.childNodes[i].matches == "undefined"
+                ? element.childNodes[i].matches(selector)
+                :element.childNodes[i].msMatchesSelector(selector)
+                )){
+                result.push(element.childNodes[i]);
+            }
+        }
+        return result;
+    };
+
+    empty(){
+        jQuery.empty(this._string);
+        return;
+    }
+
+    //Remove all child nodes of the set of matched elements from the DOM.
+    static empty(node){
+        let element = document.querySelector(node);
+        
+        while (element.firstChild){
+            let child = element.firstChild;
+            child.parentNode.removeChild( child);   
+        }        
+    };
+
+    css(property){
+        jQuery.css(this._string, property);
+        return this;
+    }
+
+    static css(node, property){
+        let element = document.querySelector(node);
+        var regex = RegExp('^[a-z\-\ ]+:[a-z\ ]+$');
+        if (!regex.test(property))
+            return window.getComputedStyle(element).getPropertyValue(property);
+        element.setAttribute("style", property);
+    };
+
+    click(callback){
+        jQuery.click(this._string, callback);
+        return this;
+    }
+
+    static click(node, callback){
+        let element = document.querySelector(node);
+        element.addEventListener('click', callback);
+    };
+
+    foreach(callback){
+        this.foreach(this._string, callback);
+        return this;
+    }
+
+    static foreach(selector, callback){
+        let elements = document.querySelectorAll(selector);
+        for(let i = 0; i < elements.length; i++){
+            callback.call(null, elements[i]);
+        }
+    }
+
+    toggle(){
+        jQuery.toggle(this._string);
+        return this;
+    }
+
+    static toggle(selector){
+        let elements = document.querySelectorAll(selector);
+        for(let i = 0; i < elements.length; i++){
+            elements[i].style.display = "none";
+        }
+    }
+    
+}
+
+jQuery.addClass('body', 'katya');
+jQuery.removeClass('body', 'katya');
+jQuery.append('ul', '<li>SMILE one</li>');
+jQuery.css('li', "background-color:blue");
+jQuery.empty('ul');
+console.log(jQuery.text('body'));
+
+console.log(jQuery.children(".demo-container", 'ul'));
+console.log(jQuery.attr(".demo-box", 'something'));
+
+console.log(jQuery.css('body', 'background-color'));
+
+
+jQuery.click('body', function() {
+    alert("addEventListener"); 
+  });
+
```

Comment:
Please, check whether chaining works. I tried to call an example from task and there are lots of errors.

Please, be sure that such simplified example works:
```
new jQuery(p')
    .foreach(function (element) {
        console.log(element);
    })
    .append('<div>Test</div>')
    .addClass('my-super-class')
    .css('backgroundColor', 'rebeccapurple');
```

Reply:
Checked, chaining is working